tests/run.sh

```diff
@@ -19,9 +19,9 @@ function bail() {
 test -n "$CONNECTOR" || bail "must specify CONNECTOR env variable"
 test -n "$VERSION" || bail "must specify VERSION env variable"
 
-FLOW_IMAGE="quay.io/estuary/flow:v0.1.0-376-g273cd3d"
+FLOW_IMAGE="ghcr.io/estuary/flow:v0.1.0-406-gebef86c"
 # the connector image needs to be available to envsubst
-export CONNECTOR_IMAGE="ghcr.io/estuary/${CONNECTOR}:${VERSION}" 
+export CONNECTOR_IMAGE="ghcr.io/estuary/${CONNECTOR}:${VERSION}"
```

Comment:
One thing we might consider doing is replacing the pair of `CONNECTOR` & `VERSION` variables with directly expecting `CONNECTOR_IMAGE` to be set. I've found myself doing local docker builds without the `ghcr.io/estuary` prefixes (eg. `source-s3:dev` rather than `ghcr.io/estuary/source-s3:dev`), but then I can't actually run the connector tests using those local images.

Reply:
I'm 👍 to `CONNECTOR_IMAGE` (why have two variables when one will do?). Not in this PR though (let's 🚢 it!).